src/main/java/org/influxdb/dto/Point.java

```diff
@@ -385,4 +387,51 @@ private StringBuilder formatedTime() {
     return sb;
   }
 
+  public Point tag(final String tagName, final String value) {
+
```

Comment:
asking twice did not fix the formatting issues, without them fixed im wont merge.